### PR TITLE
Added an option to show or not show sector info for `lotus-miner info`

### LINKED ISSUE #3913: it's very slow to called `lotus-miner info` when the sectors number is over 10000.
**Describe the bug**

it's very slow to called `lotus-miner info` when the sectors number is over 10000.

i think it cause by using too much time to count every sector status by here
https://github.com/filecoin-project/lotus/blob/4121063ca45cedb766c963f360522efa285bd23b/cmd/lotus-storage-miner/info.go#L292

### PR DESCRIPTION
**Pull request by jennijuju**
fixes #3913 (more on surface level)

Should consider implementing #4002 to fix it from root.

### CODE REVIEW

File: cmd/lotus-storage-miner/info.go [hunk 1]
```diff
@@ -33,6 +33,12 @@ var infoCmd = &cli.Command{
 	Subcommands: []*cli.Command{
 		infoAllCmd,
 	},
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "show-sectors-info",
+			Usage: "show sectors info",
```
Comment (magik6k) [inline cmd/lotus-storage-miner/info.go:39]: I'd make this flag inverted, so the default is to show the sectors (`hide-sectors`)

File: cmd/lotus-storage-miner/info.go [hunk 2]
```diff
@@ -199,10 +205,12 @@ func infoCmdAct(cctx *cli.Context) error {
 
 	fmt.Printf("Expected Seal Duration: %s\n\n", sealdur)
 
-	fmt.Println("Sectors:")
-	err = sectorsInfo(ctx, nodeApi)
-	if err != nil {
-		return err
+	if cctx.Bool("show-sectors") {
```
Comment (magik6k) [inline cmd/lotus-storage-miner/info.go:208]: Doesn't match the Name above

File: cmd/lotus-storage-miner/info.go (hunk 1 above)
Comment (jennijuju) [reply cmd/lotus-storage-miner/info.go]: updated

File: cmd/lotus-storage-miner/info.go (hunk 2 above)
Comment (jennijuju) [reply cmd/lotus-storage-miner/info.go]: 😅

**Review by magik6k — APPROVED** [review]

**Pull request merged**